src/mish2jsonl/mish2jsonl.c

```diff
@@ -0,0 +1,160 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include "mish_common.h"
+
+typedef struct m_string {
+  uint64_t len;
+  char *string;
+} m_string;
+
+typedef struct worker_output {
+  uint32_t status;   // 4
+  uint16_t ndecoded; // 2
+  uint32_t workerno; // 4
+  m_string workerso; // 16
+  m_string result;   // 16
+} worker_output;
+
+typedef struct cohort_results {
+  uint32_t nworkers;
+  m_string input;
+  worker_output *outputs;
+} cohort_results;
+
+cohort_results results;
```

Comment:
Nit: Make these `static` to indicate that they're internal to the translation unit.